CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
+        self.assertEqual(7, Calculator.run_operation(7, 10, '%'))
```

Comment:
This is a good test case, maybe add another to test out different cases such as a negative number or decimal and what would happen if a user inputted numbers like that

Reply:
I think there should be more test cases, there are plenty of age cases to cover.

Reply:
Yes, more test cases would definitely increase reliability for different inputs

Reply:
I agree! A test case for non-numbers would also be a good case to add